spec/factories/articles.rb

```diff
@@ -3,7 +3,7 @@
 
   factory :article do
     transient do
-      title { generate :title }
+      title { Faker::Food.dish }
```

Comment:
I want to point out that this attribute is not used unless `with_title` is used with it`